solara/server/flask.py

```diff
@@ -220,9 +221,20 @@ def read_root(path):
     if not allowed():
         return login()
 
+    samesite = "lax"
```

Comment:
I added some duplicate code here. What would be a good place to store this, if you like it to be refactored. Other option might be to test only the `starlette` server and don't touch the `flask` part.

Reply:
It’s ok, since we test it, unless you see an elegant way